src/main/kotlin/org/akuleshov7/utils/Logger.kt

```diff
@@ -10,6 +10,7 @@ import kotlin.system.exitProcess
 
 /**
  * Yes, it is yet another logger, because the quality and functionality of the current loggers is extremely poor
```

Comment:
I don't really see a reason for this: for JVM projects you still can use kotlin-logging as a facade and logback/log4j as an implementation.

Reply:
will make it later

Reply:
Well, I don't insist and for a small CLI tool it's probably fine to have a small custom logger, I'm just not quite convinced of advantages over existing solutions.